chain/indexer/Cargo.toml

```diff
@@ -21,6 +21,7 @@ tokio.workspace = true
 tracing.workspace = true
 
 nearcore.workspace = true
+near-async.workspace = true
```

Comment:
Can you also remove actix? Not sure if unused dependencies are detected automatically, maybe it's still used elsewhere.

Reply:
In the end we'll remove actix entirely, but right now there are places that we cannot remove yet. The indexer likely still depends on actix because of ViewClientActor that we haven't migrated. So I think it's more efficient to just not worry about this dependency until the whole migration is complete and then we'll remove it from the whole workspace.